ios/FluentUI.Demo/FluentUI.Demo/Demos/ContactCollectionViewDemoController.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class ContactCollectionViewDemoController: UIViewController {
+    private let contactCollectionView = ContactCollectionView()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = Colors.surfacePrimary
+
+        contactCollectionView.translatesAutoresizingMaskIntoConstraints = false
+        contactCollectionView.contactList = samplePersonas
+
+        view.addSubview(contactCollectionView)
+
+        NSLayoutConstraint.activate([
+            contactCollectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
```

Comment:
Could we add some top spacing above the collection? I think this would look nice.
contactCollectionView.topAnchor.constraint(equalTo: view.topAnchor constant:20)